src/components/views/rooms/MemberInfo.js

```diff
@@ -440,20 +440,35 @@ module.exports = withMatrixClient(React.createClass({
         const roomId = this.props.member.roomId;
         const target = this.props.member.userId;
         const room = this.props.matrixClient.getRoom(roomId);
-        const self = this;
-        if (!room) {
-            return;
-        }
-        const powerLevelEvent = room.currentState.getStateEvents(
-            "m.room.power_levels", "",
-        );
-        if (!powerLevelEvent) {
-            return;
-        }
+        if (!room) return;
+
+        const powerLevelEvent = room.currentState.getStateEvents("m.room.power_levels", "");
+        if (!powerLevelEvent) return;
+
         if (powerLevelEvent.getContent().users) {
```

Comment:
Probably best to check the negation of this, set the power levels and return early for the sake of less indentation.

Reply:
IMO in this case it adds more lines and not a fan but have done it now